### ISSUE TITLE
[Visual Requirements - Change Visualization->Library Panels]: Luminosity ratio is less than 3:1 for “Delete” button present under 'Library panel’ blade.

### ISSUE THREAD

**Issue by natellium** [grafana/grafana#66310]
### User Experience:  
Users with low vision might get affected as the luminosity ratio for Button control is less than 3:1 as a result users will not be able to view the button properly.

### Repro Steps:
 Open #Url: https://play.grafana.org/  Sign in with valid credentials.  "Home" page appears.  Navigate the Feature Showcase pane heading and select it. Navigate the expanded Dropdown and select the "Edit" menu item.  Navigate the page "Edit" page which appears. Navigate to the "Change Visualization" button and select it.  Navigate to the "Library Panels" tab which appears and select. Navigate the Library panel. Run CCA Tool Verify the issue.

### Actual Result: Issue: 
Luminosity ratio is less than 3:1 for “Delete” button present under 'Library panel’ blade. Expected Result:

Solution: Luminosity ratio should bemore than 3:1 for “Delete” button present under 'Library panel’ blade.
Note: The same issue exists with " focus indicator of list items under calculation combo box" under 'Table footer' section on 'Grafana play home/edit panel'. Same issue for Query patterns text as 2.2:1

### Severity
`SEV2`

**Comment by natellium** [comment]
Fixed by https://github.com/grafana/grafana/pull/69907
